guides/manual-installation.md

```diff
@@ -45,9 +49,12 @@ Enable PackageHub repository:
 SUSEConnect --product PackageHub/15.5/x86_64
 ```
 
-> Note: Using a different Service Pack then SP5 requires to change repository: [SLE15 SP3: `SUSEConnect --product PackageHub/15.3/x86_64`, SLE15 SP4: `SUSEConnect --product PackageHub/15.4/x86_64`]
+> Note: SLE15 SP3 requires a provided prometheus server. The version available through **SUSEConnect --product PackageHub/15.3/x86_64** is outdated and is not compatible with Trento's prometheus configuration.
```

Comment:
I don't understand the `SLE15 SP3 requires a provided prometheus server`. Does this mean that you need to have your own prometheus installation?
`SLE15 SP3 requires an already installed prometheus server, as the given prometheus version for this service pack is outdated`, or something similar.
Anyway, i know that that's true, but it sound terrible in a guide :sweat_smile: